addons/cargo/initSettings.inc.sqf

```diff
@@ -30,6 +30,16 @@ private _category = [ELSTRING(main,Category_Logistics), LSTRING(openMenu)];
     {[QGVAR(paradropTimeCoefficent), _this, true] call EFUNC(common,cbaSettings_settingChanged)}
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(unloadOnKilled),
+    "SLIDER",
+    [LSTRING(unloadOnKilled), LSTRING(unloadOnKilled_description)],
+    _category,
+    [0, 1, 0, 1, true], // [_min, _max, _default, _trailingDecimals, _isPercentage]
+    1,
+    {[QGVAR(unloadOnKilled), _this, true] call EFUNC(common,cbaSettings_settingChanged)}
```

Comment:
Only noticed this now: Generally speaking, do we need to keep this code?
I'm only using it for notifying people about settings needing a mission restart.

`EFUNC(common,cbaSettings_settingChanged)` raises `ace_settingChanged`, which seems to still be supported according to a post made in 2017 on ACE v3.12.0.

We might be better off calling `ace_settingChanged` from `CBA_SettingChanged`, which would handle that automatically.

Reply:
What I'll do it then is:
- modify `cbaSettings_settingChanged` to handle the warnings only
- clean up calls to `cbaSettings_settingChanged`
- raising `ace_settingChanged` will be handled by `CBA_settingChanged`